dyninstAPI/h/BPatch_snippet.h

```diff
@@ -238,8 +238,10 @@ class BPATCH_DLL_EXPORT BPatch_constExpr : public BPatch_snippet {
     //  Creates a representation of a (signed long) value
     BPatch_constExpr(signed long value);
     
-    //  Creates a representation of an (unsigned long) value
```

Comment:
Is uintptr_t portable? I think there are, broadly, three use categories:

1) We want an actual unsigned long, or a uint_32t or uint64_t, in a public interface as a numeric constant.
2) We want a Dyninst::Offset or Dyninst::Address in a public interface, reflecting a pointer or offset in the mutatee's address space.
3) We're working with internals, in which case we again typically want to be explicit about the meaning of a given alias for an unsigned long: is it a size_t, an Offset, an Address, a fixed uint_64?

In general I'm nervous about uintptr_t pervading the code unless we really mean "type of a pointer in the mutator's address space".